static/app/utils/analytics/replayAnalyticsEvents.tsx

```diff
@@ -92,6 +92,9 @@ export type ReplayEventParameters = {
     platform: string | undefined;
     project_id: string | undefined;
   };
+  'replay.render-missing-replay-alert': {
+    location: string;
```

Comment:
This might be misinterpreted as a `Location` object. Maybe it could be renamed to like `surface` or something. It's not fully a 'page' value because there are multiple spots within issue details, so which spot exactly is what i'm aiming for